source/blog/2018-05-04-the-emberjs-times-issue-45.md

```diff
@@ -1,71 +1,118 @@
+Seconds, minutes, a coffee later—you might have checked your terminal window—then at last, you find that syntax error in your template
+that led to a failing build, preventing the reload.
+
+This week's RFC proposes the automatic inclusion of **<i>ember-cli-build-notifications</i>**
+into new Ember apps. It's an addon providing system notifications for failing Ember app builds by default.
+
+The RFC proposal also discusses several alternative techniques to improve developer ergonomics regarding
+app builds. If you're curious, check out the [full RFC here](https://github.com/emberjs/rfcs/pull/328)
+and leave your thoughts on it!
+
+<a class="ember-button ember-button--centered" href="https://github.com/emberjs/rfcs/pull/328">Read more</a>
 
 ---
 
 ## [Listen to your Broccoli and it will tell you how to eat it](http://www.oligriffiths.com/broccolijs/)
 
 You might already know that Broccoli takes care of your asset pipeline. But do you know **how** it works? [Oli Griffiths](https://twitter.com/oligriffiths) has made an [amazing tutorial](http://www.oligriffiths.com/broccolijs/) on *the asset pipeline for ambitious applications*. It takes you through the basic setup and guides you through the journey of sass preprocessing, debugging and fingerprints.
 
+---
+
+## [Ember Inspector dark mode](https://github.com/emberjs/ember-inspector)
 
+![Ember Inspector Dark Mode Screenshot](/images/blog/emberjstimes/ember-inspector-dark.png)
+
+The Ember Inspector project is on a roll! v3.1.0 has the new component tree and dark mode. Change your theme from light to dark in Chrome/Firefox dev tools. 🖤
 
 ---
 
-## [EMBER DATA TITLE HERE](https://github.com/emberjs/rfcs/pull/329)
+## [Spring cleaning for Ember Data powered by RFCs 📟🐹](https://github.com/emberjs/rfcs/pull/329)
+
+A particular RFC (Request for Comments) has entered the final comment period with incredible speed.
+It proposes the **deprecation** of <em>ember-data</em>'s `Array.filter` method and explains
+how this change will not only improve the maintainability of the
+<i>ember-data</i> project but also impact the **performance** of Ember apps positively.
+
+You can [read the full RFC here](https://github.com/emberjs/rfcs/pull/326) and
+leave your comments for it.
+
+If you're making use of `this.get('store').filter` in your app or addon,
+check out the documentation of the <i>ember-data-filter</i> addon and
+follow the dedicated [guide on how to refactor this easily](https://github.com/ember-data/ember-data-filter#recommended-refactor-guide).
 
-mention final comment period of this RFC: https://github.com/emberjs/rfcs/pull/326
-and also this new RFC: https://github.com/emberjs/rfcs/pull/329
-here
+In other <i>ember-data</i> news, another proposal focuses on removing usage of the `Ember.Evented` mixin
+in <i>ember-data</i> specifically.
+This would also lead to the future removal of several lifecycle hooks and methods
+on e.g. `Model`s and other Ember Data classes. You can learn more about
```

Comment:
I wasn't sure about the mix of "_ember-data_" and "Ember Data", should we be consistent?